client/keys/add.go

```diff
@@ -112,6 +116,14 @@ func RunAddCmd(cmd *cobra.Command, args []string, kb keys.Keybase, inBuf *bufio.
 	interactive := viper.GetBool(flagInteractive)
 	showMnemonic := !viper.GetBool(flagNoBackup)
 
+	algo := keys.SigningAlgo(viper.GetString(flagKeyAlgo))
```

Comment:
Secp256k1 is already the default. Why manually do this?

Reply:
Setting `Secp256k1` as the default wasn't working for me, something weird with cobra.